Gemfile

```diff
@@ -52,6 +52,8 @@ gem "vite_rails"
 # For GOV.UK branding
 gem "govuk-components", "~> 4.1.0"
 gem "govuk_design_system_formbuilder", "~> 4.1.1"
+# The autocomplete component is not currentky published as a gem
+gem "dfe-autocomplete", require: "dfe/autocomplete", github: "DFE-Digital/dfe-autocomplete"
```

Comment:
The dfe/autocomplete component hasn't been published - like our markdown Gem. This makes it feel a little less stable. I guess we can always fork it/ inline it/ come up with something else if it turns out to be an issue.